### ISSUE TITLE
Login broken in master

### ISSUE THREAD

**Issue by Andy2244** [jellyfin/jellyfin-androidtv#1835]
### Describe the bug

I cant login via username only anymore using master?
Server is successfully via:
`Authentication request for andy has succeeded.`

Yet the login button quickly shows "invalid user/password", btw those info text are gone in like 0.5s and are nearly impossible to catch.

Than i get back to the setup screen with my user icon showing, but cant get past this screen.

Trying to figure out what change broke it, any idea?

### Application version

master

### Where did you install the app from?

Sideloaded APK

### Device information

z9x

### Android version

Android 9

### Jellyfin server version

10.8.1

**Comment by nielsvanvelzen** [comment]
Yeah that's a regression from #1826, we don't recommend running master on device as it's not been tested for production use. Should be fixed soonish.

**Comment by nielsvanvelzen** [comment]
I think it might be better to base your fork of the release branches (currently release-0.13.z, soon to be release-0.14.z). The release branches are what is released to Google Play / Amazon Appstore and is tested to be stable (and in case of an issue, those get fixed fast). The master branch is never guaranteed to be stable, in fact; during the release cycle of 0.12 it didn't work properly for multiple months.

**Comment by Andy2244** [comment]
> The master branch is never guaranteed to be stable, in fact; during the release cycle of 0.12 it didn't work properly for multiple months.

Yeah after 0.14 is released i will do this and only cherry-pick important stuff from master.
My fork is also still in beta, so users don't expect a stable version atm and i try to test all new changes from master.

**Comment by nielsvanvelzen** [comment]
Issue is now fixed on master
